src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = " ╔╦═╦══╦═╦╗\n" +
+                      " ║╚╗║╔╗║═╣║\n" +
+                      " ║╔╣║╚╝║═╣║\n" +
+                      " ║╚═╩══╩═╝║\n" +
+                      " ╚════════╝\n";
+        System.out.println("Hiiii, my name is\n" + logo );
+        System.out.println("I am here to help :3\n");
+        String line;
+        String[] tasks = new String[100];
```

Comment:
May I clarify if there is a reason why a Task Class was not created to store the array of tasks?